### ISSUE TITLE
Does it make sense to merge this with youtube-video-js somehow?

### ISSUE THREAD

**Issue by TehShrike** [npm-dom/youtube-video#10]
@mkay581 is interested in merging https://github.com/mkay581/youtube-video-js with this repo/project somehow.

Does this make sense?  What would the entry point to the web component be?  I don't feel comfortable pointing the `main` entry point to something completely different.

**Comment by markcellus** [comment]
Yeah I'd love to find some way to collaborate on this. It looks like this API is very very similar and we both have people who want both of our packages. I'm just an advocate for less fragmentation in the open-source community so I'd like to get your ideas on it.

@TehShrike the main entry point would be to the web component class which would be exported, like I'm doing [here](https://github.com/mkay581/youtube-video-js/blob/d2e9f45e439ec299d1f7147c022cc9ed08990943/src/youtube-video.ts#L11).

**Comment by TehShrike** [comment]
What I meant is: I'm not willing to change the `main` entry point of the `youtube-video` package to something that isn't a function like the one currently exported.

If the youtube-video package also contained a web component, it would have to be exposed some other way.  e.g. the readme could say "if you want this web component, you should `import { YoutubeVideoElement } from youtube-video/web-component.js`"
